app/src/main/java/com/sun/weather/di/RepositoryModule.kt

```diff
@@ -5,14 +5,18 @@ import com.sun.weather.data.repository.WeatherRepositoryImpl
 import com.sun.weather.data.repository.source.WeatherDataSource
 import com.sun.weather.data.repository.source.local.WeatherLocalDataSource
 import com.sun.weather.data.repository.source.remote.WeatherRemoteDataSource
+import org.koin.core.qualifier.named
 import org.koin.dsl.module
 
 val RepositoryModule =
     module {
         single {
             provideWeatherRepository(
                 WeatherLocalDataSource(get()),
-                WeatherRemoteDataSource(get()),
+                WeatherRemoteDataSource(
+                    baseApiService = get(named("baseApiService")),
+                    proApiService = get(named("proApiService")),
```

Comment:
```suggestion
                    baseApiService = get(named("baseApiService")),
                    proApiService = get(named("proApiService")),
```
"baseApiService" & "proApiService" dùng ở nhiều nơi mình nên tạo biến const cho nó rồi dùng thay vì hard code như này e nhé!